vllm/compilation/collective_fusion.py

```diff
@@ -212,7 +209,7 @@ def call_trtllm_fused_allreduce_norm(
                 hidden_dim=allreduce_in.shape[-1],
                 workspace_ptrs=_FI_WORKSPACE_TENSOR,
                 launch_with_pdl=launch_with_pdl,
-                use_oneshot=True,
+                use_oneshot=num_tokens <= _ONESHOT_MAX_TOKENS,
```

Comment:
@ilmarkov was it intentional to hardcode True? Perhaps this may help with the overhead issues you've seen

Reply:
Yes, two shot algorithm was slower than unfused options.

Reply:
@ilmarkov Which PR is it?